README.md

```diff
@@ -346,6 +346,77 @@ Within Flow (and Process Builder), there are 4 invocable actions that you can us
 
 ---
 
+## Tagging Your Log Entries
+
+Nebula Logger supports dynamically tagging/labeling your `LogEntry__c` records via Apex and Flow. Tags can then be stored using one of the two supported modes (discussed below).
+
+### Adding Tags in Apex
+
+// TODO LogEntryEventBuilder.addTag(String tagName) and addTags(List<String> tagNames)
```

Comment:
are these upcoming (post PR) changes?

Reply:
Oops, I overlooked these TODOs - I'm going to include these in this PR, I'll try to finish up the README changes tonight